.vscode/settings.example.json

```diff
@@ -19,6 +19,18 @@
     "**/node_modules": true,
     "dist/**": true
   },
+  "[jsonc]": {
```

Comment:
I still think this is better as settings.json for now.  Re: disabling yaml, I think that's fine for now and I think a future answer is to add a yaml formatter to the editor settings, the lint CI, and the lint-staged steps.

Reply:
This is still the sticking point for me here.  I am not convinced that moving to a separate file is worth the friction that it creates yet.  If there was more than just this one thing, maybe I would feel differently, but at this point I think having a single file that auto-updates for devs is better.